### PR TITLE
Add chipseq-greylist.

### PR DESCRIPTION
**Pull request by roryk**
This is a command line re-implementation of the ChIP-seq greylisting Bioconductor package written in python with some fixes to make it easier to use when building ChIP-seq pipelines.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by roryk** [comment]
Looks like the Travis build completed https://travis-ci.org/bioconda/bioconda-recipes/builds/299836323?utm_source=github_status&utm_medium=notification

**Comment by daler** [comment]
travis-ci had some issues with communicating with github, I wonder if that's popping up again. I'll retrigger the build.

File: recipes/chipseq-greylist/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [ `uname` == Darwin ]; then
+	export MACOSX_DEPLOYMENT_TARGET=10.9
```
Comment (druvus) [inline recipes/chipseq-greylist/build.sh]: This is included in `scripts/env_matrix.yml`so it is not needed.

**Comment by roryk** [comment]
Thanks @druvus, nuked that.

**Comment by roryk** [comment]
Thanks @druvus!

**Pull request merged**